arm-servicefabric/2016-09-01/examples/ClusterPatchOperation_exmaple.json

```diff
@@ -0,0 +1,192 @@
+        ]
+      }
+    },
+    "responses": {
+      "200": {
+        "headers": {},
+        "body": {
+          "type": "Microsoft.ServiceFabric/clusters",
+          "location": "southcentralus",
+          "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/resRg/providers/Microsoft.ServiceFabric/clusters/myCluster",
+          "name": "myCluster",
+          "tags": {
+            "resourceType": "Service Fabric",
+            "clusterName": "myCluster"
+          },
+          "etag": "W/\"636236839047996583\"",
+          "properties": {
+            "provisioningState": "Updating",
+            "clusterId": "466e7604-b8ae-4696-a91d-d7b5fcc63628",
+            "clusterCodeVersion": "5.5.149.9494",
+            "clusterState": "UpdatingUserConfiguration",
+            "managementEndpoint": "https://myCluster.southcentralus.cloudapp.azure.com:19080",
+            "clusterEndpoint": "https://warp-test-winfabrp-southcentralus.trafficmanager.net/runtime/clusters/466e7604-b8ae-4696-a91d-d7b5fcc63628",
+            "certificate": {
+              "thumbprint": "483B5DE52ECA35147B94C5E116FA7B2E34BB68F9",
+              "x509StoreName": "My"
+            },
+            "clientCertificateThumbprints": [],
+            "clientCertificateCommonNames": [],
+            "fabricSettings": [
+              {
+                "name": "Security",
+                "parameters": [
+                  {
+                    "name": "ClusterProtectionLevel",
+                    "value": "EncryptAndSign"
+                  }
+                ]
+              }
+            ],
+            "upgradeDescription": {
+              "overrideUserUpgradePolicy": false,
+              "forceRestart": true,
+              "upgradeReplicaSetCheckTimeout": "10675199.02:48:05.4775807",
+              "healthCheckWaitDuration": "00:05:00",
+              "healthCheckStableDuration": "00:05:00",
+              "healthCheckRetryTimeout": "00:45:00",
+              "upgradeTimeout": "12:00:00",
+              "upgradeDomainTimeout": "02:00:00",
+              "healthPolicy": {
+                "maxPercentUnhealthyNodes": 100,
+                "maxPercentUnhealthyApplications": 100
+              },
+              "deltaHealthPolicy": {
+                "maxPercentDeltaUnhealthyNodes": 0,
+                "maxPercentUpgradeDomainDeltaUnhealthyNodes": 0,
+                "maxPercentDeltaUnhealthyApplications": 0
+              }
+            },
+            "diagnosticsStorageAccountConfig": {
+              "storageAccountName": "ba3wzulspn67w2",
+              "protectedAccountKeyName": "StorageAccountKey1",
+              "blobEndpoint": "https://ba3wzulspn67w2.blob.core.windows.net/",
+              "queueEndpoint": "https://ba3wzulspn67w2.queue.core.windows.net/",
+              "tableEndpoint": "https://ba3wzulspn67w2.table.core.windows.net/"
+            },
+            "nodeTypes": [
+              {
+                "name": "nt1vm",
```

Comment:
value for "name" property in the PATCH payload is " "n1" but in the response it is "nt1vm"